src/stores/news.ts

```diff
@@ -0,0 +1,26 @@
+import {UGentArticle} from "../types/stores";
+import {ENDPOINTS} from "../constant";
+import {useSuspenseQuery} from "@tanstack/react-query";
+import {useFocusNotifyOnChangeProps} from "../lib/hooks/useFocusNotifyOnChangeProps";
+import {NativeModules, Platform} from "react-native";
+
+export const useUGentQuery = () => {
+  const notifyOnChangeProps = useFocusNotifyOnChangeProps();
+
+  return useSuspenseQuery<UGentArticle[], Error>({
+    queryKey: ["news"],
+    queryFn: async () => {
+
+      // This will later be managed by the settings in the app
+      var language = 'en';
+      if((Platform.OS == "android" && NativeModules.I18nManager.localeIdentifier.startsWith("nl")) 
+      || (Platform.OS == "ios" && NativeModules.SettingsManager.settings.AppleLocale.startsWith("nl"))) {
+        language = 'nl';
+      }
```

Comment:
Replace this with `https://docs.expo.dev/guides/localization/#getting-the-users-language`. If you rebase the branch on master, all dependencies should be there for you

Reply:
Done.